debian/build_virtualenv

```diff
@@ -40,6 +40,7 @@ dh_virtualenv \
     --upgrade-pip \
     --preinstall="lxml" \
     --preinstall="mock" \
+    --preinstall="wheel" \
```

Comment:
could you explain why this is needed/what benefit it brings?

Reply:
Per the [commit message](https://github.com/matrix-org/synapse/pull/11269/commits/51a7c9e0b79dc3415ebacebc525119795acc119a):

> Addresses the following warnings during Debian builds:
>
>     Using legacy 'setup.py install' for jaeger-client, since package 'wheel' is not installed.
>     Using legacy 'setup.py install' for matrix-synapse-ldap3, since package 'wheel' is not installed.
>     Using legacy 'setup.py install' for opentracing, since package 'wheel' is not installed.
>     Using legacy 'setup.py install' for psycopg2, since package 'wheel' is not installed.
>     Using legacy 'setup.py install' for systemd-python, since package 'wheel' is not installed.
>     Using legacy 'setup.py install' for pympler, since package 'wheel' is not installed.
>     Using legacy 'setup.py install' for threadloop, since package 'wheel' is not installed.
>     Using legacy 'setup.py install' for thrift, since package 'wheel' is not installed.

Reply:
ack thanks, didn't realise I should review the commit messages.

Reply:
👍 I'll explicitly call out when commit messages might be relevant in future PRs.